lib/include/tick/solver/sto_solver.h

```diff
@@ -9,7 +9,11 @@
 
 #include "tick/base/base.h"
 #include "tick/base_model/model.h"
+
+#include "tick/linear_model/model_logreg.h"
```

Comment:
Why do we need to include logreg and proxl1?

Reply:
leftovers likely - to remove